package.json

```diff
@@ -32,9 +32,20 @@
   },
   "main": "index.js",
   "dependencies": {
-    "node-sass": "3.8"
+    "node-sass": "4.6.0"
   },
   "devDependencies": {
+    "browser-sync": "^2.24.3",
```

Comment:
Might be good to remove the carets to have specific versions.

Reply:
gotcha, I'll also remove browser sync, since its not ready for prime time...